contracts/test/_fixture.js

```diff
@@ -32,7 +32,7 @@ const sfrxETHAbi = require("./abi/sfrxETH.json");
 
 async function defaultFixture() {
   await deployments.fixture(undefined, {
-    keepExistingDeployments: Boolean(isForkWithLocalNode),
+    keepExistingDeployments: true,
```

Comment:
This is not needed as env var `LOCAL_PROVIDER_URL` set to `http://localhost:8545` can be added to the launch config

Reply:
Setting this to `true` will make it work with standalone node as well as without one. That way, we don't have to force everyone to run a standalone node

Reply:
If you are actively writing contracts and the tests, you'll need the contract to be redeployed (assuming the contracts' code will keep changing). But if you don't feel strongly about this, can change it